test/studies/prk/DGEMM2/dgemm.chpl

```diff
@@ -0,0 +1,115 @@
+use Time;
```

Comment:
You should probably add a header comment similar to the others, putting your name as the contributor.